extensions/amp-social-share/0.1/storybook/Basic.amp.js

```diff
@@ -76,6 +77,8 @@ storiesOf('amp-social-share v1', module)
     const paramAttribution = text('data-param-attribution', undefined);
     const paramMedia = text('data-param-media', undefined);
     const appId = text('data-param-app_id', '254325784911610');
+    const width = text('width', undefined);
```

Comment:
I think technically these should be `null` values and not undefined.

Reply:
Thanks Dima, updated throughout!